server/restivus-swagger.js

```diff
@@ -66,12 +67,18 @@ Restivus.prototype.addSwagger = function(path) {
 
         // Add paths to Swagger doc
         _.extend(doc, {"paths": paths});
+
+        // Add definitions
+        if(swagger.definitions !== undefined) {
+          _.extend(doc, {"definitions": swagger.definitions});
+        }
+
         // Return swagger.json
         return doc;
 
       } else {
         // Error handling
-        return {"error": "Swagger metadata not given in Restivus config."};
+        return {"error": "Swagger configuration not given for Restivus."};
```

Comment:
In general, does this PR make Swagger metadata manditory for all API endpoints?
It seems like we are throwing an error if an endpoint does not contain Swagger metadata.